xxhash.h

```diff
@@ -3365,6 +3365,7 @@ XXH_FORCE_INLINE uint64x2_t XXH_vld1q_u64(void const* ptr)
 #    if defined(__GNUC__) && !defined(__APPLE_ALTIVEC__)
 #      define __APPLE_ALTIVEC__
 #    endif
+#    define vector __vector
```

Comment:
Refinement suggestion : define `vector` _only if_ it's not already defined within `<altive.h>`

Reply:
That would be a problem. I'd have to put that define AFTER including altivec.h, but then clang will exit with error.